recipes-core/images/orangepi-image-initramfs.bb

```diff
@@ -0,0 +1,23 @@
+DESCRIPTION = "Small image capable of booting the orangepi R1+"
+
+INITRAMFS_SCRIPTS ?= "\
+                      initramfs-framework-base \
+                      initramfs-module-udev \
+                     "
+
+PACKAGE_INSTALL = "${INITRAMFS_SCRIPTS} ${VIRTUAL-RUNTIME_base-utils} udev base-passwd ${ROOTFS_BOOTSTRAP_INSTALL}"
+
+# Do not pollute the initrd image with rootfs features
+IMAGE_FEATURES = ""
+
+export IMAGE_BASENAME = "${MLPREFIX}orangepi-image-initramfs"
```

Comment:
Personal Inquiry, MLPREFIX means what?

Reply:
Multilib prefix... I did not know that before right now ahah. It was inspired of core-image-minimal-initramfs. I can add that to the bb file to know where it comes from. It seems that MLPREFIX is necessary to support many variants of the image that could be built